SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -51,6 +51,111 @@ SCDocHTMLRenderer {
 		^str.replace(" ", "%20")
 	}
 
+	*replaceRegexp { |source, findRegexp, replace|
+		var founds, replaced;
+		founds = source.findRegexp(findRegexp);
+		founds = if(findRegexp[0] == $^) {
+			founds.collect { |array| if (array[0] == 0) {array} {} }
+		} {
+			founds
+		};
+		while { founds.includes(nil) } { founds.remove(nil) };
+		founds = founds.asSet.asArray.sort({ |a, b| a[0] < b[0] });
+		replaced = source;
+		if(founds.size > 0) {
+			founds.reverse.do { |idx_str|
+				var foundIndex, foundString;
+				#foundIndex, foundString = idx_str;
+				replaced = if (foundIndex > 0) {
+					var lastString = replaced[foundIndex + foundString.size ..];
+					lastString = if(lastString != nil) { lastString } { "" };
+					replaced[0 .. foundIndex - 1] ++ replace ++ lastString
+				} {
+					replace ++ replaced[foundString.size ..]
+				}
+			}
+		} {
+			replaced
+		};
+		^replaced
+	}
+	*teletypeWithinCode { |str|
+		^str
+		.replace(":teletype:", "teletype::")
+		.replace(":/teletype:", "::")
+	}
+	*parseMathML { |str|
```

Comment:
Remove this. It does not fit this PR.

Reply:
This part is related to the following line of the corresponding schelp:
- https://github.com/supercollider/supercollider/blob/12f3979d5bc19f6415e13a47e83d63dcaed269cf/HelpSource/Reference/MathJax_LaTeX_MathML.schelp#L45
- https://github.com/supercollider/supercollider/blob/12f3979d5bc19f6415e13a47e83d63dcaed269cf/HelpSource/Reference/MathJax_LaTeX_MathML.schelp#L52
- https://github.com/supercollider/supercollider/blob/12f3979d5bc19f6415e13a47e83d63dcaed269cf/HelpSource/Reference/MathJax_LaTeX_MathML.schelp#L54
- https://github.com/supercollider/supercollider/blob/12f3979d5bc19f6415e13a47e83d63dcaed269cf/HelpSource/Reference/MathJax_LaTeX_MathML.schelp#L71
- https://github.com/supercollider/supercollider/blob/12f3979d5bc19f6415e13a47e83d63dcaed269cf/HelpSource/Reference/MathJax_LaTeX_MathML.schelp#L79 
- https://github.com/supercollider/supercollider/blob/12f3979d5bc19f6415e13a47e83d63dcaed269cf/HelpSource/Reference/MathJax_LaTeX_MathML.schelp#L81

The rendered help document is as follows:
![Screenshot 2024-04-27 at 15 59 23](https://github.com/supercollider/supercollider/assets/416281/93d5e6d1-a6c0-4441-964d-05c0a9688f84)
![Screenshot 2024-04-27 at 15 59 10](https://github.com/supercollider/supercollider/assets/416281/2ae010d0-6eb4-4872-a152-d7338228d3d8)

If you still think this is unnecessary. I can remove it, of course, but could you reconsider it to make the help document more reader friendly?

Reply:
@prko are we in agreement that we won't need to render math in the style modal tags, like `code::`, `teletype::`, `emphasis::`, etc.?

Reply:
@mtmccrea Optionally, they may be needed in some cases. However, I am not sure that even I might encounter this case for my use..... Um. Ok, get rid of it!

Reply:
I removed `:teletype`